docs/local-development.md

```diff
@@ -64,7 +64,8 @@ asked to consent to share their information with the partner before being sent b
 
 To simulate a true end-to-end user experience, you can either...
 
-- Use the built-in test controller for SAML logins at http://localhost:3000/test/saml/login
+- Use the built-in test controller for SAML logins at http://localhost:3000/test/saml/login or OIDC logins at http://localhost:3000/test/oidc/login
+  Note: to update service provider configurations,  run __rake db:seed__ comand
```

Comment:
```suggestion
  Note: to update service provider configurations,  run the command `rake db:seed` or `make setup`
```

`make setup` also runs `db:seed` so I think this note could be helpful.

Reply:
@night-jellyfish , thanks, I usually only use `make update`.